tests/e2e/test_rosacli_ocm_role.go

```diff
@@ -25,6 +25,33 @@ var _ = Describe("Edit ocm role", labels.Feature.OCMRole,
 			rosaClient = rosacli.NewClient()
 			ocmResourceService = rosaClient.OCMResource
 		})
+		It("make sure on ocm-role and user role linked - [id:52106]", labels.High, labels.Runtime.Day2, func() {
```

Comment:
The steps are related to ocm-role/user-role deletion, seems a bit of conflict to the case title. Can u please clarify more?